.github/workflows/django.yml

```diff
@@ -0,0 +1,30 @@
+name: Django CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
```

Comment:
so you're testing on a bunch of python versions? Cool!

Any reason why? Why not just fix the python version?

Reply:
Roger that